Sources/Tracking/CustomerIO.swift

```diff
@@ -1,4 +1,4 @@
-import CioInternalCommon
+@_exported import CioInternalCommon
```

Comment:
This is the magic line that helps us not break customers methods, they wouldn't need to specifically add import for common for the methods they were using previously.

Reply:
I have not seen `@_exported` before so I looked it up. 

From my research thus far, it appears that this is a private/internal Swift feature not meant to be used in an iOS app. 

Sometimes this isn't a problem, sometimes it can result in people's apps getting rejected by Apple when they find non-public code in an app. 

Because our codebase does have an alternative solution that should fix this same problem, I strongly suggest that we remove this line and instead add a new line to [this file](https://github.com/customerio/customerio-ios/blob/main/Sources/Tracking/Type/Aliases.swift): 

```swift
public typealias CustomerIO = CioInternalCommon.CustomerIO
```

that way we can avoid a possible scenario where customer's apps get rejected because of our SDK.

Reply:
Absolutely, not going to advocate for the usage of it. Even though App Store guidelines do not advise the usage against it and but its meant to be used with swift and not external apps. This looks like a neat solution that is meant for this purpose. 

I can definitely try 
```
public typealias CustomerIO = CioInternalCommon.CustomerIO
```